.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build znai
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [1.8,  11]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Set up grahpviz
+        run: sudo apt-get install graphviz
+
+      - name: Build with xvfb enabled
+        uses: GabrielBB/xvfb-action@v1.0
+        with:
+          run: mvn -B package
```

Comment:
I would suggest `verify` rather than `package` in case we ever add plugins that run during `verify`.